src/AdaptAuth.ts

```diff
@@ -62,14 +62,6 @@ export class AdaptAuth {
           encodedSUID: profile.encodedSUID as string,
         };
 
-        // Attach relayState to req
-        try {
-          (req as SamlUserRequest).samlRelayState = JSON.parse(req.body.RelayState);
-        } catch (err) {
-          // I guess the relayState wasn't that great...
-          console.log('Unable to parse samlRelayState', err);
-        }
-
```

Comment:
Any particular reason for not attaching the SAML relay state to the request object? This would prevent any custom downstream middleware from accessing the SAML relay after authentication.

Reply:
I went ahead and restored it. Doesn't seem to be causing any issues.